SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -206,4 +207,13 @@ extension HomeViewController: SliderListDelegate{
         self.collectionView.reloadData()
     }
     
+    func spinnerLoadingState(state: Bool) {
```

Comment:
please put a name understandable state is not,  I would understand that if is true you should show the spinner so its not clear

Reply:
As I said before is super important.... what the function do and what is the parameter exactly do should be understandable without need to go to the function and read the code